qa/TL1_jupyter_plugins/test.sh

```diff
@@ -13,16 +13,17 @@ do_once() {
 
 test_body() {
   # attempt to run jupyter on all example notebooks
-    black_list_files="optical_flow_example.ipynb\|#" # optical flow requires TU102 architecture
-                                                     # whilst currently L1_jupyter_plugins test
-                                                     # can be run only on V100
+  black_list_files="optical_flow_example.ipynb\|tensorflow-dataset-*\|#" 
+  # optical flow requires TU102 architecture whilst currently L1_jupyter_plugins test can be run only on V100
+  # tensorflow-datset requires TF >= 1.15
```

Comment:
You can add a note that the tensorflow dataset jupyter notebooks are run in `TL1_tensorflow_dataset` job

Reply:
Done